app/views/groups/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="d-flex flex-fill bg-primary justify-content-center align-items-center text-light p-2">
+  <div class="text-center flex-grow-1">CATEGORIES</div>
+  <div class="text-center">
+    <small><%= link_to "New", new_group_path, class: 'text-decoration-none text-white' %></small>
+  </div>
+</div>
+<div id="groups">
+  <% @groups.each do |group| %>
+    <%= link_to(group, class:"text-decoration-none") do %>
+      <div class="m-3 bg-white d-flex gap-3 p-2" id="<%= dom_id group %>">
+        <div>
+          <%= image_tag group.icon, size: '50x50' %>
+        </div>
+        <div class="text-muted flex-grow-1">
+          <small class="text-decoration-none"><%= group.name %></small>
+        </div>
+        <div class="text-black fw-bold">
+          <%= number_to_currency(group.payments.sum(:amount)) %>
+        </div>
+      </div>
+    <% end %>
+  <% end %>
+</div>
```

Comment:
- [ ] Please create the big green button at the bottom of the category `index` page that allows you to add a new category. Currently, you have a link that says "New" in the navigation which does the same thing. Please remove it and create the button instead so that we stick as closely as possible to the design guidelines 👍
![image](https://user-images.githubusercontent.com/46839548/229303446-5a04df44-7af6-47e3-9c80-abedffcbffc7.png)

- [ ] Please remove the links above the navigation. Some of them like "New category" and "New payment" are redundant and are not consistent with the design guidelines 
![image](https://user-images.githubusercontent.com/46839548/229302676-02f63d27-4d84-4034-9d2e-51b189f12ded.png)

- [ ] Please add dates to each category card as shown in the design guideline below 👇 
![image](https://user-images.githubusercontent.com/46839548/229303685-8ed3bbee-d171-410b-8646-2f5017339003.png)